vocabularies/Common.xml

```diff
@@ -218,15 +218,12 @@
       </Term>
 
       <Term Name="SecondaryKey" AppliesTo="EntityType" Type="Collection(Edm.PropertyPath)" Nullable="false">
```

Comment:
One of our colleagues mentioned that it would make sense in course of this de-deprecation to call the term SecondaryKeys as it's a collection (analogous to AlternateKeys from the Core vocabulary.

Reply:
No, this is a _single_ secondary key with _one or more key properties_ - we do have lots of multi-part keys at SAP.

Multiple secondary keys are differentiated via qualifiers.